vars/mtaBuild.groovy

```diff
@@ -28,6 +31,16 @@ def call(Map parameters = [:]) {
                                       parameters, parameterKeys,
                                       stepConfiguration, stepConfigurationKeys, stepDefaults)
 
+
+        MTA_JAR_FILE_VALIDATE: {
```

Comment:
If MTA_JAR_LOCATION is in the path, the version must be validated anyways.

Reply:
I guess you mean when MTA_JAR_LOCATION is configured. It makes only little sense to put the folder represented by MTA_JAR_LOCATION in the path since the mta toolset is not an application like, ```java```, ```neo```, etc ... It is simply a jar file lauched by ```java -jar```.
If MTA_JAR_LOCATION is set (this is) we validate. I guess this is what you request above.